tests/py_test.py

```diff
@@ -89,11 +90,12 @@ def asserter(stmt):
 
```

Comment:
I think there needs to be a branch here that checks the file identifier exists, ideally using generated code.

Here is example generated code in Java:
https://github.com/google/flatbuffers/blob/master/tests/MyGame/Example/Monster.java#L16

And its test:
https://github.com/google/flatbuffers/blob/master/tests/JavaTest.java#L87